shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/NearbyStaticData.kt

```diff
@@ -572,13 +572,7 @@ fun NearbyStaticData.withRealtimeInfo(
             compareBy<StopsAssociated, Route>(Route.pinnedRoutesComparator(pinnedRoutes)) {
                     it.sortRoute()
                 }
-                .thenBy {
-                    it.patternsByStop.all { byStop ->
-                        byStop.patterns.all { patterns ->
-                            patterns.upcomingTrips.isNullOrEmpty() && !patterns.hasSchedulesToday
-                        }
-                    }
-                }
+                .thenBy { !it.hasSchedulesToday }
```

Comment:
nitpick (non-blocking): this name feels a bit of a misnomer - docs would help, maybe something like `it.ordinarilyWouldRunToday`

Reply:
Yeah that's fair. That's just the only use case we're using it in, and I don't have a great idea for a name to encompass all of these. I don't know how likely/possible it is, but I could see `ordinarilyWouldRunToday` also becoming a misnomer if there are no scheduled trips but for some reason we do start running service and have upcoming trips, then it ordinarily wouldn't be running service but abnormally are. Or if we have a disruption on a route with no weekend service, if the alert isn't specifically set to exclude the weekend, and this is true, that's not ordinarily running either. But I'm being pedantic and those are very contrived examples. 😅 Uhhh... Maybe, `hasServiceOrDisruptionToday`? Orrr.. for very vague but more literal, something like `hasNearbyDisplayPriority`?

Reply:
Yeah I see what you mean 😃  I'd be fine with either of those two, I think the doc will be more important for conveying the nuance anyway.